### DISCUSSION TITLE
Is it possible to digitize a multi-part polygon in QField (on mobile device)?

### DISCUSSION THREAD

**Discussion by cez2020** [opengisch/QField#4774] [Q&A]
I have a project that requires collecting multi-part polygon data on-site. I know how to accomplish this in QGIS but can't find a way to digitize it on QField on a mobile device. Any ideas?

**Comment by OBVYamaska** [comment]
I'm tackling a similar issue, and as far as I know, it is not possible.
I've tried many workaround, but did not succeed.
In your case, would it be possible to create different polygons, store the same ID in every feature (dropdown from existing ID after creating it with the first feature), and merge them once on QGIS?

**Comment by ZsanettMed** [comment]
Hi @cez2020 and @OBVYamaska!

Currently, QField does not support digitizing multipart polygons. However, we highly value feedback from users like you because it helps us improve and update our features. Your suggestion has been posted on our [QField Ideas Platform](https://ideas.qfield.org/app-feature-requests/p/digitize-multipart-polygon) where you and others can show their support by upvoting it. Another option is to sponsor the development of this feature. If that interests you, please send us an email at [sales@qfield.com](mailto:sales@qfield.com). 

Best regards
Zsanett
